src/west/main.py

```diff
@@ -22,7 +22,8 @@
 
 from west import log
 from west import config
-from west.commands import CommandError, CommandContextError, external_commands
+from west.commands import CommandError, CommandContextError, \
+    external_commands, BadExternalCommand
```

Comment:
Just noticed here, as the errors are lined-up side-by-side.
Should we rename, `BadExternalCommand` ?
```
CommandError
CommandContextError
BadExternalCommand !!!
```
maybe this is nicer:
```
CommandError
CommandContextError
ExternalCommandError or maybe ExternalCommandInvalidError
```
Thoughts ?
This can also be made into independent issue top be taken later.

Reply:
Sure, that's a good and easy change to make.